src/tests/functional/plugin/gna/shared_tests_instances/behavior/ov_executable_network/get_metric.cpp

```diff
@@ -188,19 +188,19 @@ INSTANTIATE_TEST_SUITE_P(
 // TODO: verify hetero interop
 INSTANTIATE_TEST_SUITE_P(DISABLED_smoke_OVClassHeteroExecutableNetworlGetMetricTest,
         OVClassHeteroExecutableNetworkGetMetricTest_SUPPORTED_CONFIG_KEYS,
-        ::testing::Values("GNA"));
+        ::testing::Values("HETERO:GNA"));
 
 // TODO: verify hetero interop
 INSTANTIATE_TEST_SUITE_P(DISABLED_smoke_OVClassHeteroExecutableNetworlGetMetricTest,
         OVClassHeteroExecutableNetworkGetMetricTest_SUPPORTED_METRICS,
-        ::testing::Values("GNA"));
+        ::testing::Values("HETERO:GNA"));
 
 // TODO: verify hetero interop
 INSTANTIATE_TEST_SUITE_P(DISABLED_smoke_OVClassHeteroExecutableNetworlGetMetricTest,
         OVClassHeteroExecutableNetworkGetMetricTest_NETWORK_NAME,
-        ::testing::Values("GNA"));
+        ::testing::Values("HETERO:GNA"));
 
 INSTANTIATE_TEST_SUITE_P(smoke_OVClassHeteroExecutableNetworlGetMetricTest,
         OVClassHeteroExecutableNetworkGetMetricTest_TARGET_FALLBACK,
-        ::testing::Values("GNA"));
+        ::testing::Values("HETERO:GNA"));
```

Comment:
Why do you change this test for GNA?

Reply:
You requested me to change the `target_device` string formation in the test

Reply:
reverted